stl/inc/random

```diff
@@ -683,12 +683,12 @@ public:
     }
 
     void seed(_Seed_t _Value = default_seed) { // set initial values from specified seed value
-        _Seed(_Value, false, true_type{});
+        _Seed<true>(_Value, false);
     }
 
     template <class _Gen>
     void seed(_Gen& _Gx, bool _Readcy = false) { // set initial values from range
-        _Seed(_Gx, _Readcy, is_arithmetic<_Gen>{});
+        _Seed<is_arithmetic<_Gen>::value>(_Gx, _Readcy);

```

Comment:
While the inline variable is technically only available in C++17 I find that it is universally available in this implementation
```suggestion
        _Seed<is_arithmetic_v<_Gen>>(_Gx, _Readcy);
```